exir/serde/serialize.py

```diff
@@ -617,12 +617,6 @@ def deserialize(
             symbol_name_to_range,
             res.names_to_symbols,
         )
-        model_opset_version: Optional[Dict[str, int]] = exported_program.opset_version
-        self._validate_model_opset_version(model_opset_version)
-
-        upgrader = GraphModuleOpUpgrader(
```

Comment:
It seems actually used, is it OK to delete it like this? Any risk?

Reply:
I don't think this is actually being used. I'll confirm with export folks